python/cog/server/http.py

```diff
@@ -53,23 +54,15 @@ def predict(request: Request = Body(default=None)):
         """
         Run a single prediction on the model.
         """
-        if request is None or request.input is None:
-            output = predictor.predict()
-        else:
-            try:
+        has_input = request is not None and request.input is not None
```

Comment:
Nit: this logic might be a bit neater if `request.input` was assigned to a variable `input`, which was by default `None`? (or a better name which isn't a global, haha)

Reply:
Nope no opinions, just an idea!